src/components/analytics/metrics-table.vue

```diff
@@ -133,6 +132,16 @@ export default {
       "num_offline_entity_submissions_reprocessed": "Number of reordered Offline Entity submissions",
       "max_entity_submission_delay": "Maximum Entity Submission processing delay",
       "avg_entity_submission_delay": "Average Entity Submission processing delay",
+      "uses_external_blob_store": "Server uses external blob store",
```

Comment:
I think we should avoid the word "blob", since most translators won't know what that is. From a quick look at the user docs, I see the terms "S3-compatible storage" and "external object store". How about using one of those? This comment also applies to some other metrics below that use the word "blob".